src/types/form.types.ts

```diff
@@ -57,7 +56,7 @@ export const basicSchema = z
         (value) => TestPlatformOptions.some((option) => option.value === value),
         'Invalid option selected'
       ),
-    name: z.string({ required_error: 'This field is required' }),
+    name: z.string({ required_error: 'This field is required' }).min(1, 'This field is required'),
```

Comment:
whats this doing

Reply:
I was a bug. 
we want name as mandatory field but empty string was validated previously. so I added a check that it must have atleast 1 char in input.